nextflow_schema.json

```diff
@@ -380,13 +372,6 @@
             "fa_icon": "fas fa-dna",
             "description": "Options for the reference genome indices used to align reads.",
             "properties": {
-                "igenomes_base": {
-                    "type": "string",
-                    "description": "Directory / URL base for iGenomes references.",
-                    "default": "s3://ngi-igenomes/igenomes/",
-                    "fa_icon": "fas fa-cloud-download-alt",
-                    "hidden": true
-                },
                 "igenomes_ignore": {
                     "type": "boolean",
```

Comment:
Can delete

Reply:
ah ok, now I got it I though it was about igenomes_base which was already deleted

Reply:
the parameter group "reference_genome_options" will be emtpy though... and if I remove the whole parameter group linting fails.